src/apps/answers/api.py

```diff
@@ -634,3 +636,23 @@ async def answers_existence_check(
     )
 
     return Response[AnswerExistenceResponse](result=AnswerExistenceResponse(exists=is_exist))
+
+
+async def applet_submissions_list(
+    applet_id: uuid.UUID,
+    user: User = Depends(get_current_user),
+    query_params: QueryParams = Depends(parse_query_params(PaginationQueryParams)),
+    session=Depends(get_session),
+    answer_session=Depends(get_answer_session),
+) -> PublicSubmissionsResponse:
+    await AppletService(session, user.id).exist_by_id(applet_id)
+    await CheckAccessService(session, user.id).check_answer_access(applet_id)
+    submissions, submissions_count = await AnswerService(session, user.id, answer_session).get_applet_submissions(
+        applet_id, query_params
+    )
+
+    participants_count = await WorkspaceService(session, user.id).get_workspace_applet_respondents_total(applet_id)
```

Comment:
I find it strange to include `participants_count` here as it's somewhat unrelated to `submissions` which is the point of this endpoint.
From the conversation in Jira, it sounds like `participants_count` was meant to be added to the `applets/:applet_id/respondents` endpoint (in `workspace_applet_respondents_list`)

Reply:
Participant count is already being returned by the `applets/{applet_id}/respondents` endpoint isn't it? No need to add there, nor add to this submissions endpoint.